src/software/thunderscope/robot_communication.py

```diff
@@ -209,6 +209,13 @@ def setup_for_fullsystem(self):
             True,
         )
 
+        self.receive_ssl_referee_proto = SSLRefereeProtoListener(
+            SSL_REFEREE_ADDRESS,
+            SSL_REFEREE_PORT,
+            lambda data: self.current_proto_unix_io.send_proto(Referee, data),
+            True,
+        )
```

Comment:
Added a UDP listener for referee messages coming from the game controller to robot communication. We did have this at robocup last year (code was copied from `robot_diagnostics2` branch), but it appears that we missed it when we merged that branch. Currently on master we only create a referee listener when we're running simulation and did not create one if we're running just fullsystem (`run_blue`, `run_yellow`).